nservicebus/messaging/databus/index.md

```diff
@@ -95,16 +95,17 @@ Automatically removing these attachments can cause problems in many situations.
 
 ## Alternatives
 
-- Use a different transport or a different transport tier
-- Message Compression: Use message body compression which works well on text-based payloads like XML and Json or any payload (text or binary) that contains repetitive data
-  - [Message mutator example demonstrating message body compression](/samples/messagemutators/)
-- Stream-based properties: The [Handling large stream properties via pipeline](/samples/pipeline/stream-properties/) sample demonstrates a purely stream-based approach (rather than loading the full payload into memory) implemented by leveraging the NServiceBus pipeline.
-- Binary Serializer: A binary serializer is more efficient and most serializers can be added with a few lines of code
-   - Some binary [serializers are maintained by the community](/nservicebus/community/#serializers)
-- Attachments: When dealing with unbounded binary payloads consider the [community maintained NServiceBus.Attachments](/nservicebus/community/#nservicebus-attachments)
-  - Read on demand: Will only retrieve attachment data when the consumer reads it
-  - Reduced Memory usage: No base64 serializer overhead resulting in a significant reduction in resource utilization
-- Use any of the above in combination with compression
+NOTE: A combination of these techniques may be used.
+
+- Use a different transport or different tier (e.g. Azure Service Bus _Premium_ instead of _Standard_).
+- Message compression: this works well on text-based payloads like XML and JSON, or any payload (text or binary) that contains repetitive data.
```

Comment:
```suggestion
- Message compression: This works well on text-based payloads like XML and JSON, or any payload (text or binary) that contains repetitive data.
```

Reply:
I think it should stay as-is. It's arguable that the text after the colon is not a complete sentence since it relies on the "this" context from before the colon, and most style have it optional anyway for complete sentences. I would say it's better to be consistent with lists and partial sentences and start with lower case.